charts/argocd-progressive-rollout/README.md

```diff
@@ -28,9 +28,11 @@ Users of Helm v2 should use `--set installCRDs=true` to install the CRDs.
 | affinity | object | `{}` |  |
 | args.enableLeaderElection | bool | `false` |  |
 | args.metricsAddr | string | `":8080"` |  |
-| configSecret.argoCDAuthToken | string | `"example-token"` |  |
-| configSecret.argoCDInsecure | string | `"true"` |  |
-| configSecret.argoCDServerAddr | string | `"argocd-server"` |  |
+| config.argoCDAuthToken | string | `"example-token"` |  |
+| config.argoCDInsecure | string | `"true"` |  |
+| config.argoCDServerAddr | string | `"argocd-server"` |  |
+| configSecret.annotations | object | `{}` |  |
+| configSecret.name | string | `""` |  |
```

Comment:
Should we add some documentation here as to what this does? either here or in the main readme?
just mentioning somewhere that if `configSecret.name` is not specified, the helm chart will generate a secret and use it.

Reply:
This parameter is often called `existingSecret` in many other charts that I have used.

Reply:
I pushed a change to add a description